libewf/libewf_single_file_permission.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * Single file source functions
+ *
+ */
+
+#if !defined( _LIBEWF_SINGLE_FILE_PERMISSION_H )
+#define _LIBEWF_SINGLE_FILE_PERMISSION_H
+
+#include <common.h>
+#include <types.h>
+
+#include "libewf_date_time.h"
+#include "libewf_libcerror.h"
+
+#if defined( __cplusplus )
+extern "C" {
+#endif
+
+typedef struct libewf_single_file_permission libewf_single_file_permission_t;
+
+struct libewf_single_file_permission
+{
+	/* The id
+	 */
+	uint32_t id;
+
+	/* The type
+	 */
+	uint8_t type;
+
+	/* The name
+	 */
+	uint8_t *name;
+
+	/* The name size
+	 */
+	size_t name_size;
+
+	/* The uid
+	 */
+	uint8_t *uid;
```

Comment:
@jtsylve could you confirm that this value is not only used for a user identifier (uid) but also for a group identifier (gid) if the (permission) type is not owner?

Reply:
I was able to confirm this myself, uid is is not the most fitting name. I'll go for (ACE) identifier.